From 817ba9d5227c5aa83e24ad7414d60aaf5bf3e047 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Mon, 24 Jan 2022 14:48:06 +0000
Subject: [PATCH 92/94] WIP: arm64: dts: meson: add SSV6051 wifi to Tanix
 TX3-mini

Testing an experimental out-of-tree ssv6xx driver [0]

[0] https://github.com/paolosabatino/ssv6xxx

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxl-s905w-tx3-mini.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxl-s905w-tx3-mini.dts b/arch/arm64/boot/dts/amlogic/meson-gxl-s905w-tx3-mini.dts
index 6705c2082a78..b013a7934e4d 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxl-s905w-tx3-mini.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxl-s905w-tx3-mini.dts
@@ -25,6 +25,13 @@ &ir {
 	linux,rc-map-name = "rc-tanix-tx3mini";
 };
 
+&sd_emmc_a {
+	/* SSV6051 */
+	ssv: wifi@1 {
+		reg = <1>;
+	};
+};
+
 &usb {
 	dr_mode = "host";
 };
-- 
2.35.1

